File: app/src/main/java/com/wlmtest/forecasttestjava/model/fivedaysforecastdata/Coord.java

package com.wlmtest.forecasttestjava.model.fivedaysforecastdata;

import com.squareup.moshi.Json;

public class Coord {

    @Json(name = "lat")
    private Double lat;
    @Json(name = "lon")
    private Double lon;

    public Double getLat() {
        return lat;
    }

    public void setLat(Double lat) {
        this.lat = lat;
    }

    public Double getLon() {
        return lon;
    }

    public void setLon(Double lon) {
        this.lon = lon;
    }

}

File: app/src/main/java/com/wlmtest/forecasttestjava/model/api/fivedaysforecastdata/Coord.kt
package com.wlmtest.forecasttestjava.model.api.fivedaysforecastdata

import com.squareup.moshi.Json

class Coord {

    @Json(name = "lat")
    var lat: Double? = null
    @Json(name = "lon")
    var lon: Double? = null

}
